content/05.methods.md

```diff
@@ -192,28 +192,31 @@ We also explored the performance of vg and SMRT-SV v2 in different sets of regio
 ### Yeast graph analysis
 
 For the analysis of graphs from *de novo* assemblies, we utilized publicly available PacBio-derived assemblies and Illumina short read sequencing datasets for 12 yeast strains from two related clades (Table {@tbl:strains}) [@doi:10.1038/ng.3847].
-We selected five strains for graph contruction (two from different subclades of each clade plus the reference *S.c. S288C*): *S.c. SK1*, *S.c. YPS128*, *S.p. CBS432*, *S.p. UFRJ50816*, and *S.c. S288C*.
-We constructed two different genome graphs from the assemblies of the five selected strains.
+We constructed graphs from two different strain sets:
+For the *five strains set*, we selected five strains for graph contruction (two from different subclades of each clade plus the reference *S.c. S288C*): *S.c. SK1*, *S.c. YPS128*, *S.p. CBS432*, *S.p. UFRJ50816*, and *S.c. S288C*.
+[//]: # (How were the strains selected? Randomly?)
```

Comment:
That was one of Jonas' comments that should be easy to add. Were they chosen because of some features of the assembly (quality?) or just randomly?

Reply:
The two strains from each clade were chosen from different corners of the phylogenetic tree but otherwise randomly. I will add this to the text.